Source/URLEncodedFormEncoder.swift

```diff
@@ -48,15 +48,19 @@ public final class URLEncodedFormEncoder {
         case brackets
         /// No brackets are appended to the key and the key is encoded as is.
         case noBrackets
+        /// Brackets containing the item index appended. This matches the jQuery and Node.js behavior.
+        case indexInBrackets
 
         /// Encodes the key according to the encoding.
         ///
-        /// - Parameter key: The `key` to encode.
-        /// - Returns:       The encoded key.
-        func encode(_ key: String) -> String {
+        /// - Parameter key:    The `key` to encode.
```

Comment:
Now that there are multiple parameters, you can regenerate these docs using `Command - Option - /` and get a `Parameters` section instead. Also, though it was missing before, our general style is to put an empty line between the parameters and the return.